### PR TITLE
Upkeep

### PR DESCRIPTION
**Pull request by dunkenwg**

### CODE REVIEW

File: .github/workflows/R-CMD-check.yaml [hunk 1]
```diff
@@ -25,92 +20,33 @@ jobs:
...
+          - {os: macos-latest,   r: 'release'}
           - {os: windows-latest, r: 'release'}
-          - {os: ubuntu-latest, r: 'devel', rspm: "https://packagemanager.rstudio.com/cran/__linux__/focal/latest"}
-          - {os: ubuntu-latest, r: 'release', rspm: "https://packagemanager.rstudio.com/cran/__linux__/focal/latest"}
-          - {os: ubuntu-latest, r: 'oldrel', rspm: "https://packagemanager.rstudio.com/cran/__linux__/focal/latest"}
-          - {os: ubuntu-latest, r: '3.6', rspm: "https://packagemanager.rstudio.com/cran/__linux__/focal/latest"}
-          - {os: ubuntu-latest, r: '3.5', rspm: "https://packagemanager.rstudio.com/cran/__linux__/focal/latest"}
+          - {os: ubuntu-latest,   r: 'devel', http-user-agent: 'release'}
+          - {os: ubuntu-latest,   r: 'release'}
+          - {os: ubuntu-latest,   r: 'oldrel-1'}
```
Comment (joethorley) [inline .github/workflows/R-CMD-check.yaml]: The description file says that we go back to 3.5 but we are only testing back to 4.3. We should test back futher and/or update the DESCRIPTION R version

**Review by joethorley — CHANGES_REQUESTED** [review]

File: .github/workflows/R-CMD-check.yaml (hunk 1 above)
Comment (dunkenwg) [reply .github/workflows/R-CMD-check.yaml]: I updated the R version in the DESCRIPTION as the actions are more current now

**Comment by joethorley** [comment]
@dunkenwg - can you make it test back to R 4.0?

**Comment by dunkenwg** [comment]
@joethorley, R-CMD-checks are passing now

**Review by joethorley — APPROVED** [review]
Great

**Pull request merged**